opentelemetry-sdk/benches/metric.rs

```diff
@@ -67,13 +128,115 @@ fn bench_counter(view: Option<Box<dyn View>>) -> (Context, SharedReader, Counter
 }
 
 fn counters(c: &mut Criterion) {
-    let (cx, _, cntr) = bench_counter(None);
+    let (cx, _, cntr) = bench_counter(None, "cumulative");
+    let (cx2, _, cntr2) = bench_counter(None, "delta");
 
     let mut group = c.benchmark_group("Counter");
     group.bench_function("AddNoAttrs", |b| b.iter(|| cntr.add(&cx, 1, &[])));
+    group.bench_function("AddNoAttrsDelta", |b| b.iter(|| cntr2.add(&cx, 1, &[])));
+
     group.bench_function("AddOneAttr", |b| {
         b.iter(|| cntr.add(&cx, 1, &[KeyValue::new("K", "V")]))
     });
+    group.bench_function("AddOneAttrDelta", |b| {
+        b.iter(|| cntr2.add(&cx2, 1, &[KeyValue::new("K", "V")]))
+    });
+    group.bench_function("AddThreeAttr", |b| {
+        b.iter(|| {
+            cntr.add(
+                &cx,
+                1,
+                &[
+                    KeyValue::new("K", "V"),
+                    KeyValue::new("K2", "V2"),
+                    KeyValue::new("K3", "V3"),
+                ],
+            )
+        })
+    });
+    group.bench_function("AddThreeAttrDelta", |b| {
+        b.iter(|| {
+            cntr2.add(
+                &cx2,
+                1,
+                &[
+                    KeyValue::new("K", "V"),
```

Comment:
I believe this will result in the same attributes being repeatedly used ? Suggest to use different values for the dimension. (A lot of cost in Metric SDKs would be sorting/finding the right metric stream to aggregate this to, so unless we use different dimensions, it'll be aggregating to produce a single metric point)

Reply:
Done.